.pipelines/mdnc/azure-cns-cni.yaml

```diff
@@ -96,7 +96,7 @@ spec:
           effect: NoSchedule
       initContainers:
         - name: init-cni-dropgz
-          image: "acnpublic.azurecr.io/containernetworking/mdnc-cni-dropgz"
+          image: "acnpublic.azurecr.io/containernetworking/mdnc-cni-dropgz" # CNI 1.4.35
```

Comment:
yeah that's just a reminder of what's currently tracked by that custom package (which is used to track OAI current versions). In the future ideally we'll likely just update the whole image pointing to the official dropgz release (as I did for 1.5.4).

Reply:
Let me know if this still requires changes...

Reply:
you have currently not specified a tag for this image. you will (usually, depending on the client and registry implementations) get `acnpublic.azurecr.io/containernetworking/mdnc-cni-dropgz:latest`. 

`latest` can unexpectedly change. sometimes, pushing an image without a tag will update `latest`. sometimes the registry may automatically retag `latest` to be the last explicitly pushed tag by timestamp. 

it is better practice to be explicit about the tag.

Reply:
Added tag

Reply:
I think what @rbtr  says is that, by not mentioning any tag you might defualt to `latest` and `latest` is not always the right one and it could get updated unintentionally and hence using a specific version tag might be better.

Reply:
@rbtr please take a look

Reply:
@thomasricci general PR SOP feedback: please let the reviewer resolve their comments once _they've_ determined that you've adequately addressed the issues.

@isaac-dasan is correct. by not specifying a tag you get `latest`. but even by specifying `:latest`....you get `latest`. `latest` is a dynamic tag _and it is expected to change_. You should use a specific tag that is not latest.